Frontend/kalvians_management_system-kms/src/Pages/Login.jsx

```diff
@@ -1,9 +1,91 @@
-import React from 'react'
+import { Box, Button, TextField, Typography } from '@mui/material';
+import React from 'react';
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+
 
 const Login = () => {
+
+  const [email,setEmail]=useState("");
+  const [password,setPassword]=useState("");
+
+  const Navigate = useNavigate();
+
+  const handleLogin = async() => {
+
+   if( !email || !password ){
+      alert("Please enter your details");
+      return;
+   }
+
+   
+   let res = await fetch('http://localhost:3000/registration_data',{

```

Comment:
Add domain details `http://localhost:3000` to .env file and use process.env value here.